.github/workflows/_build_rosetta.yaml

```diff
@@ -140,27 +140,27 @@ jobs:
           target: final
           build-args: |
             BASE_IMAGE=${{ steps.defaults.outputs.BASE_IMAGE }}
-
+            
       - name: Generate sitrep
         if: success() || failure()
         shell: bash -x -e {0}
         run: |
           # bring in utility functions
           source .github/workflows/scripts/to_json.sh
 
-          badge_label='${{ inputs.BASE_LIBRARY }} ${{ inputs.ARCHITECTURE }} build'
```

Comment:
I think we shouldn't introduce `Rosetta` in the `badge_label=` since we made the product decision a while back to just offer/advertise "t5x" where "upstream-t5x" is the internal package in case we need that. So for consistency, my opinion is to remove it from the summary as well.

Also our badges are pretty minimal right now:
![image](https://github.com/NVIDIA/JAX-Toolbox/assets/7576060/c1e9e310-d75c-4824-91b1-f61ef9cb7e54)
Assuming we introduce two badges (one for arm one for amd), perhaps we should have the badge label just be `${{ inputs.ARCHITECTURE }}` and show both the arm/amd build badges on the frontpage readme. I'm thinking this full name might actually be redundant.

I feel that each stages badge_label shouldn't contain the "base library" or "framework" since that will be part of the badge json endpoint filename. Does that align with how you thought of sitreps & badges @yhtang ?

Reply:
In other words, it is OK for the JSON endpoint file to contain `"label": "T5X nightly build"`, and while we place the badge in the T5X row and the build column, we will replace the label with `https://img.shields.io/endpoint?label=nightly...`

Reply:
Oh cool, I didn't know you could override, that voids my suggestion.

And just to address the initial question, `BASE_LIBRARY` doesn't encode the information of rosetta or upstream. Putting "Rosetta T5x" and "Upstream T5x" in the badge_label/summary sounds okay to me since we wouldn't be surfacing that in the front-page readme